src/app/core/parameters/pointArray.ts

```diff
@@ -0,0 +1,30 @@
+import Parameter from "../parameter";
+import Registry from "../registry";
+
+export default class PointArray extends Parameter {
+    typeString: string = "PointArray";
+    description: string = "Should be an array of points";
+
+    constructor(value: Array<Array<number>>) {
+        super("PointArray",value);
+    }
+
+    isValid(value: any) {
+        if (value instanceof Array) {
+            if (value.length == 0) {
+                return true;
+            }
+            /*for (var i in value) {
+                let paramType = Registry.registeredParams["Point"];
```

Comment:
Instead of pulling this from registry, pull import the "point.ts" and uncomment the check loop here.